### ISSUE TITLE
Incompatible with 10.6

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mattscotthudson/soundflower#32]
```
What steps will reproduce the problem?
1. Install 1.4.3
2. Soundflower.kext incompatible with operating syst
3.

What is the expected output? What do you see instead?
Popup message stating kext is incompatible with operating system and to
remove it.

What version of the product are you using? On what operating system?
1.4.3  
10.6

Please provide any additional information below.


```

Original issue reported on code.google.com by `excitene...@gmail.com` on 30 Aug 2009 at 10:40

**Comment by GoogleCodeExporter** [comment]
```
I can (sadly) confirm this and am having the same problems
```

Original comment by `ben.wood...@gmail.com` on 31 Aug 2009 at 9:18

Attachments:
- [Screen shot 2009-08-31 at 23.17.19.png](https://storage.googleapis.com/google-code-attachments/soundflower/issue-32/comment-1/Screen shot 2009-08-31 at 23.17.19.png)

**Comment by GoogleCodeExporter** [comment]
```
OS 10.6 Support is available in SoundFlower 1.5.0, now available here:
http://code.google.com/p/soundflower/downloads/list
```

Original comment by `74obje...@gmail.com` on 1 Sep 2009 at 6:20
- Changed state: **Done**
